client/app/components/Input/__tests__/InputTag.spec.jsx

```diff
@@ -9,13 +9,9 @@ describe('InputTag', () => {
     const wrapper = mount(component);
     const id = `#${InputMocks.inputTagProps.checkboxes[1].id}`;
     const value = { value: 'Two' };
-    // Hack: Looks like react-autocomplete is looking for target, but we should use currentTarget
-    wrapper.find('.tagAutocomplete').prop('onChange')({
-      target: value,
-      currentTarget: value,
-    });
-    wrapper.find('.tagAutocomplete').simulate('focus');
     expect(wrapper.find('.tagMenu').exists()).toEqual(true);
+    wrapper.find('.tagAutocomplete').simulate('change', { target: value });
```

Comment:
So this is the main change that has been made to the tests. This provides the text input (.tagAutocomplete') with the value which in this case is 'Two'. This replaces the below code:

```
    wrapper.find('.tagAutocomplete').prop('onChange')({	
      target: value,	
      currentTarget: value,	
    });	
    wrapper.find('.tagAutocomplete').simulate('focus');
```
This code causes the tests to fail because react-autosuggest doesn't render the suggestion based on the value, hence find('.tagLabel') can't find anything because there's no suggestion. 

In essence the changes I made do the same thing as before, once the input changes to 'Two' the onChange method is fired and react-autosuggest provides the appropriate suggestion.

Let me know what you think!

Reply:
This change works and is an improvement actually! :) Thanks for the update.